tests/test_transformation/test_yeojohnson_transformer.py

```diff
@@ -1,6 +1,7 @@
 import pandas as pd
 import pytest
 from sklearn.exceptions import NotFittedError
+import numpy as np
```

Comment:
we follow pep8 for imports. this should be at the top. 

You could use isort: `isort path/to/this/file on the command line`.

More info here: https://feature-engine.trainindata.com/en/latest/contribute/contribute_code.html#test-code-style

Reply:
by running 

`isort feature_engine/transformation/yeojohnson.py`

I got this:

```python
from typing import List, Optional, Union

import numpy as np
import pandas as pd
import scipy.stats as stats

from feature_engine._base_transformers.base_numerical import \
    BaseNumericalTransformer
from feature_engine._docstrings.fit_attributes import (
    _feature_names_in_docstring, _n_features_in_docstring,
    _variables_attribute_docstring)
from feature_engine._docstrings.init_parameters.all_trasnformers import \
    _variables_numerical_docstring
from feature_engine._docstrings.methods import _fit_transform_docstring
from feature_engine._docstrings.substitute import Substitution
from feature_engine.variable_handling._init_parameter_checks import \
    _check_init_parameter_variables
```

Is it OK like that?

Reply:
I don't think so, we don't use \

Reply:
Sorry I was misconfusing the files, and I apply the isort on the normal file and not on the tests
So just fix the import in the tests, now it's all okay.

Also I remove the \ from the yehojonson.py file.